README.md

```diff
@@ -89,7 +100,8 @@ entities (no need to draw an ERD):
 
 Your answer:
 ```
-Replace this with your answer
+An ERD is a visual tool used to describe relationships between data entities
+A genie and a lamp have a 1-1 relationship.  So, a genie has one lamp, and a lamp has one genie. A person could (if lucky!) have many lamps, and therefore many genies. A person could also have many pets, but a pet typically only has 1 "person" (owner). Therefore the relationship of person and pet would be 1 -  many.
 ```
```

Comment:
:+1: An ERD also describes the attributes of each entity. "Visual tool" is the key, and you got that dead on!